app/views/console/databases/index.phtml

```diff
@@ -16,9 +16,9 @@
                 data-service="databases.list"
                 data-event="load,databases.create,databases.update,databases.delete"
                 data-param-search="{{router.params.search}}"
-                data-param-limit="<?php echo APP_PAGING_LIMIT; ?>"
-                data-param-offset="{{router.params.offset}}"
-                data-param-order-type="ASC"
+                data-param-queries="orderAsc(''),limit(<?php echo APP_PAGING_LIMIT; ?>),offset({{router.params.offset|orZero}})"
```

Comment:
We could replace `orZero` with the existing `pageCurrent` filter

Reply:
Are you sure? Formula there is `Math.ceil(parseInt($value || 0) / env.PAGING_LIMIT) + 1;`
With limit of 12 and value of 5, it gives 2. I think there is more logic to that filter.

All I need here is to have 0 if the value is missing.

Reply:
My mistake, looks likes (unsurprisingly) that's used to display the current number 😅